modules/backlogs/lib/open_project/backlogs/hooks/layout_hook.rb

```diff
@@ -86,7 +86,7 @@ def controller_work_package_new_after_save(context = {})
         end
 
         if params[:copy_tasks]
-          params[:copy_tasks] += ':' if !/:/.match?(params[:copy_tasks])
+          params[:copy_tasks] += ':' if !params[:copy_tasks].include?(':')
```

Comment:
And then the Rails/NegateInclude cop will suggest that `!foo.include?('bar')` could be replaced by `foo.exclude?('bar')` :)